ajax/libs/dayjs/package.json

```diff
@@ -14,14 +14,14 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/xx45/dayjs.git"
+    "url": "https://github.com/iamkun/dayjs.git"
   },
   "npmName": "dayjs",
   "npmFileMap": [
     {
-      "basePath": "dist",
+      "basePath": "",
       "files": [
-        "**/*"
+        "dayjs.*"
```

Comment:
Suppose users need the files under `locale` and `plugins` to extend the functionality of day.js.
You can use [the pattern](https://github.com/cdnjs/cdnjs/blob/b7fbff2ad7f9021f5a9f09357a5061a61d031e13/ajax/libs/dayjs/package.json#L24-L26) you once used :)

If you did the package.json changes locally (with Git commands),
let's separate auto-update config update and repo url update, thanks!